### ISSUE TITLE
"Backed up first" sort to list more recently backed up apps first

### ISSUE THREAD

**Issue by Terrance** [MuntashirAkon/AppManager#1366]
### Please check before submitting an issue

- [X] I am using the latest version of App Manager
- [X] I have searched the issues and haven't found anything relevant
- [X] I have read the docs

### Describe a description of the new feature

Currently, if you enable Sort > Backed up first, apps are effectively grouped into "backed up" and "not backed up", then sorted alphabetically.  It's hard to use this list to find apps that have a backup but from a long time ago, and therefore could do with an updated one -- I have to scan through the "Latest backup" text on each entry.

### Describe the solution you'd like

It would be useful, either as a separate sort or a replacement to this one, if the "backed up" section was sorted by the date when the most recent backup was taken (so that the list goes from most recently backed up, to least recently backed up, to not backed up at all).

### Describe alternatives you've considered

A filter for "not backed up in the last $time" could also work, but would probably require some form of UI to select the amount of time, which doesn't fit so well with the simplicity of checkbox sort/filter options.

**Comment by MuntashirAkon** [comment]
I don't think it makes sense to add a sorting option for it since this is a very specific case. But the filtering actually make sense which is part of #321. In fact, you can already do that in the Finder page which is available in the debug builds only (it's still pre-alpha).
